src/components/ButtonTabs/ButtonTabs.scss

```diff
@@ -3,15 +3,35 @@
 
 $block: '.#{$ns}button-tabs';
 
-#{$block} {
-    margin-bottom: $indentS;
+@mixin button-text-color($color, $hoverColor: $color) {
+    &,
+    &:link,
+    &:visited,
+    &:active,
+    &:focus {
+        color: $color;
+    }
+
+    &:hover {
+        color: $hoverColor;
+    }
+}
 
+#{$block} {
     &__item {
         margin-right: $indentXXXS;
         margin-bottom: $indentXXS;
 
-        &_active {
+        --yc-button-background-color: var(--pc-tab-item-background-color);
+        --yc-button-background-color-hover: var(--pc-tab-item-background-color-hover);
+        @include button-text-color(var(--pc-tab-item-color));
+
+        &.yc-button_selected {
```

Comment:
It's not save to use classes from other libraries. Why you removed modifier `_active`?

Reply:
The lib has predefined functionality for the behavior we need. I do not see a reason to implement a custom one.

Reply:
I've just gave you a reason, it is class from other lib, it can change className someday. We try not to set styles for other libs classNames

Reply:
OK, I see your reasons. Returned the class.